kythe/java/com/google/devtools/kythe/analyzers/java/JavaEntrySets.java

```diff
@@ -278,7 +278,8 @@ public EntrySet newWildcardNodeAndEmit(JCTree.JCWildcard wild, String sourcePath
         config.getGenericsStructure().equals(JavaIndexerConfig.GenericsStructure.TPARAM)
             ? NodeKind.TVAR
             : NodeKind.ABS_VAR;
-    return emitAndReturn(newNode(kind).addSignatureSalt(sourcePath + counter));
+    return emitAndReturn(
+        newNode(kind).addSignatureSalt(sourcePath + counter).setCorpusPath(defaultCorpusPath()));
```

Comment:
Is this feasible to test?

Reply:
Done, see the verifier assertions in the DefaultCorpus.java test.